.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/ruby
 
 # Generated js bundles
 /app/assets/javascripts/generated/*
+
+# Rubymine/IntelliJ
+.idea
```

Comment:
I sort of believe that .ruby-version should generally not be ignored and should be in EVERY project that specifies the ruby version in Gemfile.

I think the reasoning for globally excluding it is so that when we work on Gems, we can test for multiple ruby versions.